### ISSUE TITLE
Equation/Figure/Table references look visually awful.

### ISSUE THREAD

**Issue by Abhi-1U** [Abhi-1U/texor#35]
Instead of the usual way of referencing an Equation/Figure/Table with their numbers, the labels are representing it.

This could be due to lack of Rmarkdown style references.

**Comment by Abhi-1U** [comment]
After a few investigations and looking up, I have found that there are instances where authors have multiple labels in the same object for example in a figure environment, there might be two labels, on in caption and other in the figure environment itself, for example :
```
\begin{figure}
\includegraphics{example_paper_files/figure-latex/plot-example-1} \caption{\label{fig:example}My Caption}\label{fig:plot-example}
\end{figure}
```
This will confuse the `texor` package and it will pick one label during conversion and if there are any references to the other label it will have ??? in it as it is not picked up during conversion.

**Comment by Abhi-1U** [comment]
A function to find and match multiple figure references within the environments is added in this commit : 0b0ad24150bfde4fc95a486eeb7d260db220f20c

TO-DO : use the stream editor to find and replace un-used references.

**Comment by Abhi-1U** [comment]
After multiple updates to the texor package, pandoc and rjtools, most environments like figure and table are labelled quite well.
Some Equations are having trouble as it is not natively converted by pandoc.

**Comment by Abhi-1U** [comment]
These problems have been fixed with version 1.1.0
